contrib/opentimelineio_contrib/application_plugins/rv/example_otio_reader/otio_reader.py

```diff
@@ -300,6 +355,64 @@ def _create_movieproc(time_range, kind="blank"):
     return movieproc
 
 
+def _add_source_bounds(media_ref, src, context):
+    if not hasattr(media_ref, "available_image_bounds"):
+        return
+
+    global_scale = context.get('global_scale')
+    global_translate = context.get('global_translate')
+    if not global_scale or not global_translate:
+        return
+
+    bounds = media_ref.available_image_bounds
+    if not bounds:
+        return
+
+    # A width of 1.0 in RV means draw to the aspect ratio, so scale the
+    # width by the inverse of the aspect ratio
+    #
+    media_info = commands.sourceMediaInfo(src)
+    height = media_info['height']
+    aspect_ratio = 1.0 if height == 0 else media_info['width'] / height
```

Comment:
Should this check if height is less than some epsilon?  Or just a hard check against the 0 value here.

Reply:
This is probably my c++ paranoia again -- checking that something is non-zero before diving by it is just a reflex.  Even when there is no media, RV's error case is 1280x720, so I'm not sure we could ever fall into a case where the height is zero (or effectively close enough to be zero).